docs/reuse/apps/app-install-v2.md

```diff
@@ -0,0 +1,16 @@
+To install the app, do the following:
+
+1. From the Sumo Logic navigation, select **App Catalog**.
+1. In the **Search Apps** field, search for and then select your app.
+1. Optionally, you can scroll down to preview the dashboards included with the app. 
+1. To install the app, click on **Install App**. 
+1. Click on **Next** in **Setup Data** section.
+1. In the **Configure** section of respective app, complete the following fields.
+    1. **Key.** Select either of these options for the data source.
+        * Choose **Source Category**, and select a source category from the list for **Default Value**.
+        * Choose **Custom**, and enter a custom metadata field. Insert its value in **Default Value**.
+1. Click on **Next**. You will be redirected to **Preview & Done** section.
+
+Your app will be installed in **Installed Apps** folder and Panels of dashboard will start to fill automatically.
+
+Each panel slowly fills with data matching the time range query and received since the panel was created. Results will not immediately be available, updating with full graphs and charts over time.
```

Comment:
```suggestion
1. From the Sumo Logic navigation, select **App Catalog**.
1. In the **Search Apps** field, search for and then select your app. <br/>Optionally, you can scroll down to preview the dashboards included with the app. 
1. To install the app, click **Install App**. 
1. Click **Next** in the **Setup Data** section.
1. In the **Configure** section of the respective app, complete the following fields.
    1. **Key.** Select either of these options for the data source.
        * Choose **Source Category**, and select a source category from the list for **Default Value**.
        * Choose **Custom**, and enter a custom metadata field. Insert its value in **Default Value**.
1. Click **Next**. You will be redirected to the **Preview & Done** section.

Your app will be installed in the **Installed Apps** folder and dashboard panels will start to fill automatically.

Each panel slowly fills with data matching the time range query and received since the panel was created. Results will not immediately be available, updating with full graphs and charts over time.
```